.travis.yml

```diff
@@ -218,6 +218,17 @@ matrix:
         sources: ['ubuntu-toolchain-r-test']
         packages: ['g++-9', 'ninja-build']
 
+  - os: linux
```

Comment:
We already have a test case above that sets `CXXFLAGS=-std=c++2a`. Maybe this can be reused.

Reply:
Unfortunately, It seems that `CXXFLAGS=-std=c++1z` didn't work under c++17.

Reply:
`CXXFLAGS=-std...` is overriden by compile feature `cxx_std_11` in top-level `CMakeLists.txt` (and by `CMAKE_CXX_STANDARD`, if it is present; see [comment](https://github.com/nlohmann/json/pull/2117#issuecomment-676597609) in #2117)